cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js

```diff
@@ -50,6 +50,20 @@ function getPaymentCoreInfo(plan) {
     };
 }
 
+/**
+ * Returns the amount and rate of a credit properties for a given plan
+ * @param  {Object} plan any alma plan
+ * @returns {Object} with the properties translated
+ */
+function getCreditInfo(plan) {
+    var costOfCredit = parseInt(plan.customer_total_cost_amount / 100, 10).toString() + '.' + (plan.customer_total_cost_amount % 100) + '€';
```

Comment:
it is the merchant responsibility to not load Alma if currency is not euro ?

Reply:
I should use `formatCurrency` which is a SFCC function to do that, you're right :), will fix

But if our merchant is not using EUR, our psp config, inside SFCC will not allow alma to be displayed.
If a merchant edit this config, yes, Alma won't work properly.